### PR TITLE
feat(REST): add the Realtime message API document

### PR DESCRIPTION
**Pull request by lupintheforth**
translating the realtime message API document

### CODE REVIEW

**Comment by leeyeh** [comment]
Deploy preview for *lc-docs-en* ready!

Built with commit ee7c01181d90ca1da72b8c8bade76e2f89416144

https://deploy-preview-138--lc-docs-en.netlify.com

File: views/desktop.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
+realtime_rest_api_v2.md=@realtime_rest_api_v2,0
```
Comment (weakish) [inline views/desktop.ini]: 这个文件要去掉

File: templates/pages/index.html [hunk 2]
```diff
@@ -137,6 +137,25 @@ <h3>Guides</h3>
...
+                <li><a href="rest_api.html#Push_Notification">Notification Push REST API</a></li>
+                <li><a href="rest_sms_api.html"> SMS REST API</a></li>
+              </ul>
+            </div>
+        <!-- end of .section-content -->
+      </div>
+      <!-- end of .row(nested) -->
+    </div>
+
+
```
Comment (weakish) [inline templates/pages/index.html]: 这些先注释掉吧，因为 merge 了就发布，用户会碰到 404

File: views/realtime_rest_api_v2.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Realtime Message REST API User Guide
+
+## Request Format
+
+For POST and PUT request, the main body of the request should be JSON. The Content-Type need configured as `application/json`.
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: request -> requests
main body -> body

File: views/realtime_rest_api_v2.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Realtime Message REST API User Guide
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: 就叫 realtime-rest-api.md 好了，因为国际版的是新用户，很多旧版的内容可以直接省略，直接向他们介绍当前版本的内容。

File: views/realtime_rest_api_v2.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+For POST and PUT request, the main body of the request should be JSON. The Content-Type need configured as `application/json`.
+
+## Authentication
+
+Authentication of the request is via the key-value pair included in the HTTP Header, the parameters are as follows:
+
+Key|Value|Meaning|Source
+---|----|---|---
+`X-LC-Id`|{{appid}}| App Id of the Current Application| Check in Control Panel -> Settings -> App keys
+`X-LC-Key`|{{appkey}}|App key of the Current Application | Check in Control Panel -> Settings -> App keys
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: Control Panel -> Dashboard

（因为之前拂尘都翻译成 dashboard，所以统一下）

File: views/realtime_rest_api_v2.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+Part of the administrative interfaces require master key.
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: Part of -> Some

File: views/realtime_rest_api_v2.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+## Relevant Concept
+
+`_Conversation` table included some embedded key strings to define the attributes and participants of the conversations. One-on-One chatting/Group Chats , Chat Rooms , Official Accounts and bots  are in this table [LeanMessage Overview - Conversation](./realtime_v2.html#conversation).
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: included -> includes
embedded key strings -> built-in fields
One-on-One 这一句空格有些问题，「Group Chats , Chat Rooms , 」 -> 「Group Chats, Chat Rooms,」，「bots  are」之间是一个空格。
[LeanMessage Overview - Conversation] -> See [LeanMessage Overview - Conversation](...) for more information.

File: views/realtime_rest_api_v2.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+## New Features
+
+With regard to the concept of 「Conversation」 , we have categorized it into three subclasses:
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: 「」-> ""

File: views/realtime_rest_api_v2.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+- One-on-One Chatting/Group Chats, related API is labelled with `rtm/conversations`.
+- Chat Rooms, related API is labelled with `rtm/chatrooms`. In `_Conversation` table, `tr` is short for true.
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: `tr` is short for true. -> `tr` is set to true to indicate this.
下句 sys 同理。

File: views/realtime_rest_api_v2.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+- Official Accounts and bots, related API is labelled with `rtm/service-conversations`. In `-Conversation` table , `sys` is short for true.
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: -Conversation -> _Conversation

File: views/realtime_rest_api_v2.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+
+In addition to that, the client related request is labelled with `rtm/clients`.
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: In addition to that -> In addition
request is -> requests are

File: views/realtime_rest_api_v2.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+## New Features
+
+With regard to the concept of 「Conversation」 , we have categorized it into three subclasses:
+
+- One-on-One Chatting/Group Chats, related API is labelled with `rtm/conversations`.
+- Chat Rooms, related API is labelled with `rtm/chatrooms`. In `_Conversation` table, `tr` is short for true.
+- Official Accounts and bots, related API is labelled with `rtm/service-conversations`. In `-Conversation` table , `sys` is short for true.
+
+In addition to that, the client related request is labelled with `rtm/clients`.  
+Eventually, some [Global API] (#Global API) are labelled with `rtm/{function}`, like `rtm/all-conversations` will search for conversations of all types.
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: Eventually -> Lastly
`[Global API] (#Global API)` -> `[global APIs](#Global_API)`

**Review by weakish — CHANGES_REQUESTED** [review]

File: views/realtime_rest_api_v2.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+`X-LC-Id`|{{appid}}| App Id of the Current Application| Check in Control Panel -> Settings -> App keys
+`X-LC-Key`|{{appkey}}|App key of the Current Application | Check in Control Panel -> Settings -> App keys
+
+Part of the administrative interfaces require master key.
+
+## Relevant Concept
+
+`_Conversation` table included some embedded key strings to define the attributes and participants of the conversations. One-on-One chatting/Group Chats , Chat Rooms , Official Accounts and bots  are in this table [LeanMessage Overview - Conversation](./realtime_v2.html#conversation).
+
+## New Features
```
Comment (weakish) [inline views/realtime_rest_api_v2.md]: New Features -> Architecture

因为中文文档是对比和 v1 的不同，这里不提 v1 所以这段标题也相应更改下。

**Review by weakish — APPROVED** [review]
还有两个 typo，我先 approve 了，合并了再改

**Pull request merged**